library/reminder/src/BirthdayReminder.php

```diff
@@ -49,11 +49,9 @@ public function isDisplayBirthdayAlert()
         }
         $today=date('Y-m-d');
         $dobStr =strtotime($res['DOB']);
-        if (
-            (($GLOBALS['patient_birthday_alert'] == 3) && ($today >= date('Y-m-d', $dobStr)) && ($today <= date('Y-m-d', strtotime('+28 days',$res['DOB']))))||
+        if ((($GLOBALS['patient_birthday_alert'] == 3) && ($today >= date('Y-m-d', $dobStr)) && ($today <= date('Y-m-d', strtotime('+28 days', $res['DOB']))))||
```

Comment:
It looks like this setting (3) is not listed in globals.inc.php . currently just have:
```
                '0' => xl('No alert'),
                '1' => xl('Alert only on birthday'),
                '2' => xl('Alert on and after birthday')
```

Reply:
disregard, i was only looking at this 1 commit. I see you made this change on this PR already :)